config/owasp/suppressions.xml

```diff
@@ -42,6 +42,20 @@
     <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-crypto@.*$</packageUrl>
     <vulnerabilityName>CVE-2020-5408</vulnerabilityName>
   </suppress>
+  <suppress>
```

Comment:
implementation group: 'org.springframework.security', name: 'spring-security-crypto', version: '5.7.5'

Reply:
That is right the updated version is 5.7.5. I left it at 5.7.3 as it is the version the cve errors flagged up.  Updated.

Reply:
I see. I add it in gradle instead.